docs/installation.md

```diff
@@ -2,8 +2,128 @@
+
+### PII anonymization on text
+
+For PII anonymization on text, install the `presidio-analyzer` and `presidio-anonymizer` packages:
+
+```sh
+pip install presidio_analyzer
+pip install presidio_anonymizer
+
+# Presidio analyzer requires a spaCy language model.
+python -m spacy download en_core_web_lg
+```
+
+For a more detailed installation of each package, refer to the specific documentation:
+
+- [presidio-analyzer](analyzer/index.md).
+- [presidio-anonymizer](anonymizer/index.md).
+
+### PII anonymization on images
+
+For PII anonymization on images, install the `presidio-image-anonymizer` package:
+
+```sh
+pip install presidio_image_anonymizer
+
+# Presidio image anonymizer uses the presidio-analyzer which requires a spaCy language model:
+python -m spacy download en_core_web_lg
+```
+
+[Click here](image-anonymizer/index.md) for more information on the presidio-image-anonymizer package.
+
 ## Using Docker
+
+Presidio can expose REST endpoints for each service using Flask and Docker. To download the Presidio Docker containers, run the following command:
+
+> This requires Docker to be installed. [Download Docker](https://docs.docker.com/get-docker/).
+
+### For PII anonymization in text
+
+For PII detection and anonymization in text, the `presidio-analyzer` and `presidio-anonymizer` modules are required.
+
+```sh
+# Download images
+docker pull mcr.microsoft.com/presidio-analyzer
+docker pull mcr.microsoft.com/presidio-anonymizer
+
+# Run containers with default ports
+docker run -d -p 5001:5001 mcr.microsoft.com/presidio-analyzer:latest
+
+docker run -d -p 5002:5001 mcr.microsoft.com/presidio-anonymizer:latest
+```
+
+### For PII anonymization in images
+
+For PII detection in images, the `presidio-image-anonymizer` is required.
+
+```sh
+# Download image
+docker pull mcr.microsoft.com/presidio-image-anonymizer
+
+# Run container with the default port
+docker run -d -p 5003:5001 mcr.microsoft.com/presidio-image-anonymizer:latest
+```
+
+Once the services are running, their APIs are available. API reference and example calls can be found [here](api.md).
+
+TODO: Update docs with local build scripts.
+
+## Install from source
+
+To install Presidio from source, first clone the repo:
+
+- using HTTPS
+
+```sh
+git clone https://github.com/microsoft/presidio.git
+```
+
+- Using SSH
+
+```sh
+git clone git@github.com:microsoft/presidio.git
+```
+
+Then, build the containers locally.
+
+> Presidio uses [docker-compose](https://docs.docker.com/compose/) to manage the different Presidio containers.
+
+From the root folder of the repo:
+
+```sh
+docker-compose --build
+```
+
+To run all Presidio services:
+
+```sh
+docker-compose up -d
+```
+
+Alternatively, you can build and run individual services. For example, for the `presidio-anonymizer` service:
+
+```sh
+docker build . -t presidio/presidio-anonymizer
```

Comment:
This will work from within the presidio-anonymizer directory. 
Either add a cd 
Or docker build ./presidio-anonymizer/Dockerfile -t presidio/presidio-anonymizer from root
Or docker-compose --build presidio-anonymizer

Reply:
Fixed, thanks!